hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/index.md

```diff
@@ -1709,6 +1709,23 @@ the storage class you want.
 Please note that S3A does not support reading from archive storage classes at the moment.
 `AccessDeniedException` with InvalidObjectState will be thrown if you're trying to do so.
 
+## <a name="upload"></a>Configuring S3A for S3 on Outposts
+
+S3A now supports S3 on Outposts, and is available in Hadoop v.3.3.7 and beyond.  AWS Outposts is a fully managed service that offers the same AWS infrastructure, AWS services, APIs, and tools to virtually any data center, co-location space, or on-premises. For more information, see the [AWS Outposts User Guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/S3onOutposts.html). Amazon S3 on Outposts delivers object storage to your on-premises AWS Outposts rack environment.  S3 on Outposts provides a new storage class, OUTPOSTS,  and you communicate with your Outposts bucket by using an access point and endpoint connection over a virtual private cloud (VPC). For more information see the [S3 on Outposts User Guide](https://docs.aws.amazon.com/AmazonS3/latest/userguide/S3onOutposts.html).
```

Comment:
I'd prefer if this was broken down to separate lines, at least after every sentence. makes it easier to review changes